platform/viewer/cypress/support/commands.js

```diff
@@ -23,3 +25,17 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('openStudy', (patientName) => {
+    cy.visit('/');
+    cy.get('#patientName')
```

Comment:
I think we're okay to start with this, but we'll want to refactor at some point to follow Cypress's best practices for selecting elements:

https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
I agree. We'll need to add `data-*` attribute on the elements.